packages/node_modules/nav-frontend-core/README.md

```diff
@@ -0,0 +1,23 @@
+# NAV-frontend-core
+
+Variabler og andre felleselementer for NAV-frontend. 
+
+## Git-historikk før februar 2019
+
+Denne pakken ble flyttet til `nav-frontend-moduler` i februar 2019. Se [tidligere eget repo](https://github.com/navikt/nav-frontend-core) for git-historikk frem til da.
```

Comment:
Forøvrig bør vi gjøre det tydelig i `README` på https://github.com/navikt/nav-frontend-core at det repoet er deprecated, og antageligvis også arkivere (https://help.github.com/en/articles/archiving-repositories).

Reply:
Absolutt 👍